README.md

```diff
@@ -247,6 +257,15 @@ In [Node.js](https://nodejs.org/en/):
 const XRegExp = require('xregexp');
 ```
 
+## Contribution Guide
+
+1. Fork the repository and clone the forked version locally.
+1. Ensure you have the `typescript` module installed globally.
+1. Run `npm install`.
+1. Ensure all tests pass with `npm t`.
+1. Add tests for new functionality or that fail from the bug not fixed.
+1. Implement functionality or bug fix to pass the test.
+
```

Comment:
Added this for two reasons:
- I generally find contribution guides helpful for new repos
- Wanted to highlight how `npm install` didn't work out of the box for me as it assumed I had `typescript` installed globally.

Happy to remove it since it's out of scope. But wanted to point it out in case it's something you guys wanted to add in the near future